### DISCUSSION TITLE
3D generation

### DISCUSSION THREAD

**Discussion by kittygg** [neperfepx/neper#808] [Q&A] [answered]
When generating a 3D model, do adjacent grians share a common surface? Can we get them to share the same surface?

**Comment by rquey** [comment, marked as answer]
Yes, they do.

**Reply by kittygg** [reply]
OK, How do I make it？

**Reply by kittygg** [reply]
The surface ID of cells show that they do not share the same surface.

**Reply by rquey** [reply]
Trust me, they do (always)...

As an example,
```
neper -T -n 2 -statcell facelist
```

returns (content of `n2-id1.stcell`):
```
1 2 3 4 5 6
7 8 9 10 11 12 6
```
which shows that face 6 is shared by the two cells.
